### PR TITLE
docs: Updating kubescape repo links

### PR DESCRIPTION
**Pull request by dwertent**
Signed-off-by: David Wertenteil <dwertent@armosec.io>

The Kubscape project has been moved from the [armosec](https://github.com/armosec) org to the [kubescape](https://github.com/kubescape) org.

In this PR I updated the relevant URLs.

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -1266,12 +1266,12 @@ integrations:
     software:
       - kubescape
     code:
-      - https://github.com/armosec/kubescape
-      - https://github.com/armosec/regolibrary
+      - https://github.com/kubescape/kubescape
+      - https://github.com/kubescape/regolibrary
     inventors:
-      - armo
+      - ARMO
```
Comment (anderseknert) [inline docs/website/data/integrations.yaml:1272]: This should be lowercased, as it is a reference/key in the `organizations` object further down in the file.

**Review by anderseknert — COMMENTED** [review]
Files in the `vendor` directory should not be edited manually — they are built by the `go mod vendor` command. If you want that changed it would need to be done in the upstream project, and then updated in OPA once they have released a new version.

File: vendor/github.com/spf13/cobra/projects_using_cobra.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
 - [Istio](https://istio.io)
 - [Kool](https://github.com/kool-dev/kool)
 - [Kubernetes](https://kubernetes.io/)
-- [Kubescape](https://github.com/armosec/kubescape)
```
Comment (srenatus) [inline vendor/github.com/spf13/cobra/projects_using_cobra.md:28]: To change this, you'd have to do it here: https://github.com/spf13/cobra/blob/main/projects_using_cobra.md?plain=1#L29, and bump the spf13/cobra dependency when a new release thereof is out.

Comment (dwertent) [reply vendor/github.com/spf13/cobra/projects_using_cobra.md]: Sorry, I did not notice this was in the vendor dir, thank you for pointing it out.

**Review by srenatus — APPROVED** [review]
It looks like all the previous links properly redirected. But if you prefer it this way, fine! 👍

**Comment by dwertent** [comment]
Thank you.
@srenatus Because we are submitting this project to CNCF we do not want any mentions of the original organization :)

**Pull request merged**